vm-setup/roles/v1aX_integration_test/vars/main.yml

```diff
@@ -7,7 +7,9 @@ CLUSTER_APIENDPOINT_HOST: "{{ lookup('env', 'CLUSTER_APIENDPOINT_HOST') }}"
 CLUSTER_APIENDPOINT_IP: "{{ lookup('env', 'CLUSTER_APIENDPOINT_IP') }}"
 CLUSTER_NAME: "{{ lookup('env', 'CLUSTER_NAME') | default('test1', true) }}"
 KUBECONFIG_PATH: "/home/ubuntu/.kube/config"
-KUBERNETES_VERSION: "{{ lookup('env', 'KUBERNETES_VERSION') | default('v1.18.0', true) }}"
+KUBERNETES_VERSION: "{{ lookup('env', 'KUBERNETES_VERSION') | default('v1.17.0', true) }}"
+KUBERNETES_BINARIES_VERSION: "{{ lookup('env', 'KUBERNETES_BINARIES_VERSION') | default(lookup('env', 'KUBERNETES_VERSION' | replace('v','')) | default('1.17.0', true) )}}"
```

Comment:
There might be an issue in the first default (parenthesis on the lookup). Maybe it could be :
```
default(lookup('env', 'KUBERNETES_VERSION') | default('v1.17.0', true) | replace('v','') , true)
```

Reply:
Done